tests/console/yast2_i.pm

```diff
@@ -98,8 +98,7 @@ sub run() {
     clear_console;                           # clear screen to see that second update does not do any more
     assert_script_run("rpm -e $pkgname");    # erase $pkgname
     script_run("echo mark yast test", 0);    # avoid zpper needle
-    script_run("rpm -q $pkgname",     0);
-    assert_screen("yast-package-$pkgname-not-installed", 1);
+    assert_script_run("rpm -q $pkgname || true");
```

Comment:
<a href="http://photobucket.com/images/confused%20cat" target="_blank"><img src="http://i1033.photobucket.com/albums/a419/lol_girl7/lol%20animals/notright.jpg" border="0" alt="confused cat photo: confused notright.jpg"/></a>

Obviously I had some "bash blackout" today … fixed

I know what it means when you are being nice to me ;-)